org-roam-refile.el

```diff
@@ -30,9 +30,70 @@
 ;; Org-roam refile allows you to refile notes to your nodes.
 ;;
 ;;; Code:
+(eval-when-compile
+  (require 'org-roam-macs)
+  (require 'org-macs))
+
 (defvar org-auto-align-tags)
 (defvar org-loop-over-headlines-in-active-region)
 
+(defun org-roam--file-keyword-get (keyword)
+  "Pull a keyword setting from the top of the file."
+  (nth 1
```

Comment:
Use `car` instead of `nth 1`

Reply:
You mean `cadr`. The `nth` function starts counting from zero.